src/storage/src/source/mod.rs

```diff
@@ -497,6 +491,7 @@ pub enum SourceMessageType<Key, Value, Diff> {
 
 /// Source-agnostic wrapper for messages. Each source must implement a
 /// conversion to Message.
+#[derive(Clone, Serialize, Deserialize)]
```

Comment:
clone for `: timely::Data` right? why the serialize and deserialize? is that just for an imagined world with exchanging across processes?

Reply:
wait, we have the rc wrapping, is this clone still required?

Reply:
oops, this is leftover from an in-progress impl, removing!